components/scream/src/physics/share/physics_test_data.hpp

```diff
@@ -40,23 +40,23 @@ struct SHOCGridData : public PhysicsTestData {
 
 // Convenience macros for up to 11 arguments, beyond that, you're on your own :)
 
-#define PTD_ZEROES0
-#define PTD_ZEROES1 0
-#define PTD_ZEROES2 PTD_ZEROES1, 0
-#define PTD_ZEROES3 PTD_ZEROES2, 0
-#define PTD_ZEROES4 PTD_ZEROES3, 0
-#define PTD_ZEROES5 PTD_ZEROES4, 0
-#define PTD_ZEROES6 PTD_ZEROES5, 0
-#define PTD_ZEROES7 PTD_ZEROES6, 0
-#define PTD_ZEROES8 PTD_ZEROES7, 0
-#define PTD_ZEROES9 PTD_ZEROES8, 0
-#define PTD_ZEROES10 PTD_ZEROES9, 0
-#define PTD_ZEROES11 PTD_ZEROES10, 0
-
-#define PTD_ZEROES(a) PTD_ZEROES##a
+#define PTD_ONES0
```

Comment:
We just want to initialize the Data struct with garbage, it's going to be assigned in the next line anyway. 1 garbage is less likely to be a problem since some of the Data constructors have inv_* members that divide.